master/internal/command/notebook_manager.go

```diff
@@ -184,31 +181,36 @@ func (n *notebookManager) newNotebook(params *CommandParams) (*command, error) {
 		return nil, errors.Wrap(err, "generating notebook config")
 	}
 
-	return &command{
-		taskID:    taskID,
-		config:    *config,
-		userFiles: params.UserFiles,
-		additionalFiles: archive.Archive{
-			params.AgentUserGroup.OwnedArchiveItem(jupyterDir, nil, 0700, tar.TypeDir),
-			params.AgentUserGroup.OwnedArchiveItem(jupyterConfigDir, nil, 0700, tar.TypeDir),
-			params.AgentUserGroup.OwnedArchiveItem(jupyterDataDir, nil, 0700, tar.TypeDir),
-			params.AgentUserGroup.OwnedArchiveItem(jupyterRuntimeDir, nil, 0700, tar.TypeDir),
-			params.AgentUserGroup.OwnedArchiveItem(
+	taskSpec := params.TaskSpec
+	taskSpec.SetInner(&tasks.StartCommand{
+		Config:    *config,
+		UserFiles: params.UserFiles,
+		AdditionalFiles: archive.Archive{
```

Comment:
do you think it makes sense to break `StartCommand` into `StartNotebook`, `Start...` and push this into that?